app/controllers/admins/employees_controller.rb

```diff
@@ -33,6 +33,25 @@ def update
       render :edit, notice: e.message
     end
 
+    def add_available_kudos
+      if params[:number_of_additional_kudos].to_i.between? 1, 20
+        @employees = Employee.all
+        begin
+          ActiveRecord::Base.transaction do
+            @employees.each do |employee|
+              employee.number_of_available_kudos += params[:number_of_additional_kudos].to_i
+              employee.save!
```

Comment:
```suggestion
              employee.update!(number_of_available_kudos: params[:number_of_additional_kudos].to_i)
```
We can edit object attributes and save them via the `update` method. Thank's to that we can write this in one line. But both ways are OK 💪🏻

Reply:
Thanks for this suggestion 🙂 I decided to change it to:

`employee.update!(number_of_available_kudos: employee.number_of_available_kudos + params[:number_of_additional_kudos].to_i)`